MoltenVK/MoltenVK/GPUObjects/MVKSwapchain.mm

```diff
@@ -33,6 +33,13 @@
 #	include <UIKit/UIScreen.h>
 #endif
 
+#if MVK_MACOS
```

Comment:
```suggestion
#if MVK_MACOS && !MVK_MACCAT
```
This is the reason for the build failure on Catalyst.

Reply:
Done! Lets see if the CI likes it.